pkg/sqlconn/config.go

```diff
@@ -72,9 +61,21 @@ func (c *config) validate() error {
 		return fmt.Errorf("sql open function must be set")
 	}
 
-	if c.connMaxIdle < 1 {
+	if c.connMaxIdleCount < 1 {
 		return fmt.Errorf("database pool max idle connections must be greater than 0")
 	}
 
+	if c.connMaxIdleTime < 1*time.Second {
+		return fmt.Errorf("database connect retry interval must be greater than 1s")
+	}
+
+	if c.connMaxLifetime < 1*time.Second {
+		return fmt.Errorf("database connection max lifetime must be greater than 1s")
+	}
+
+	if c.connMaxOpenCount < 1 {
+		return fmt.Errorf("database pool max open connections must be greater than 0")
+	}
```

Comment:
The conditions here are "less than" but in the error message is only "greater".
Please update the error message to "at least".